src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,9 @@
+package duke.exception;
+
+public class DukeException extends Exception {
+
+    @Override
+    public String toString() {
+        return "☹ OOPS!!!";

```

Comment:
Avoid magic strings. Consider declaring this as a constant `MESSAGE_ERROR` or something.